packages/MarkdownEditor-Tests.package/MarkdownInlineParserTest.class/instance/testAttributizeText.st

```diff
@@ -1,10 +0,0 @@
-tests - attributes
-testAttributizeText
```

Comment:
We removed a test about `attributizeText` as this should be covered by testing `privateStyle`.